app/controllers/pet_applications_controller.rb

```diff
@@ -37,6 +37,15 @@ def update
     redirect_back(fallback_location: root_path)
   end
 
+  def destroy
+    revoked_pet = Pet.find(params[:pet_id])
+
+    revoked_pet.update(adopt_status: "Adoptable")
+
+    flash[:approve_application] = "#{Pet.find(params[:pet_id]).name}'s application is revoked!"
```

Comment:
I think you're actively fixing this, but these messages should match up with out predefined styles